backend/src/govsg/models/govsg-template.ts

```diff
@@ -71,5 +71,5 @@ export class GovsgTemplate extends Model<GovsgTemplate> {
     allowNull: true,
     defaultValue: [],
   })
-  multilingualSupport: GovsgTemplateLanguageMetadata[]
+  multilingualSupport: Array<GovsgTemplateLanguageMetadata>
```

Comment:
I prefer using [] but I got to be consistent with the rest of the codebase as much as I see.